protos/flyteidl/external/external_event.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package flyteidl.external;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/external";
+
+import "flyteidl/core/execution.proto";
+import "flyteidl/core/identifier.proto";
+import "flyteidl/core/catalog.proto";
+import "flyteidl/core/workflow_closure.proto";
+import "flyteidl/core/compiler.proto";
+import "flyteidl/admin/event.proto";
+import "flyteidl/admin/common.proto";
+import "flyteidl/admin/execution.proto";
+import "flyteidl/admin/node_execution.proto";
+import "flyteidl/admin/task_execution.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+message WorkflowEventClosure {
```

Comment:
Should this be WorkflowEvent of ExecutionEvent closure? 
@katrogan / @EngHabu a quick look at this might help

Reply:
@katrogan but maybe its not too late to start moving away from workflow for all executions?

Reply:
that was the original goal (see https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/execution.proto) happy to just use ExecutionEventClosure